src/dotnet/UI.Blazor.App/Components/ChatView/Items/ChatMessageView.razor

```diff
@@ -193,9 +194,16 @@
                     }
                 </div>
             }
-            @if (message.ShowLinkPreview) {
-                <div class="message-link-preview">
-                    <LinkPreviewView Entry="@entry" />
+            @if (message.Entry.LinkPreviews.Count > 0) {
```

Comment:
you can just use entry (instead of message.Entry) here and below for conciseness